validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/BlobsetSequences.java

```diff
@@ -92,14 +92,15 @@ public void endpoint_config_connection_success_reconnect() {
     deviceConfig.blobset.blobs = new HashMap<>();
     deviceConfig.blobset.blobs.put(SystemBlobsets.IOT_ENDPOINT_CONFIG.value(), config);
 
-    untilTrue("blobset entry config status is success", () -> {
+    untilTrue("blobset phase is FINAL and stateStatus is null", () -> {
       BlobPhase phase = deviceState.blobset.blobs.get(
           SystemBlobsets.IOT_ENDPOINT_CONFIG.value()).phase;
-      Entry stateStatus = deviceState.system.status;
+      // Successful reconnect sends a state message with empty Entry.
+      Entry stateStatus = deviceState.blobset.blobs.get(
```

Comment:
nit: probably blobStatus is a better variable name, to differentiate from the state-system-status